modules/toast/react/lib/Toast.tsx

```diff
@@ -97,13 +97,24 @@ export default class Toast extends React.Component<ToastProps> {
         padding={PopupPadding.s}
         handleClose={onClose}
         closeIconSize={IconButtonSize.Small}
+        role={!onClose && !onActionClick ? 'status' : 'dialog'}
```

Comment:
Nit: What do you think about turning this check into a variable so it's not duplicated?

```tsx
const isInteractive = onClose && onActionClick;

return (
      <Popup
        width={toastWidth}
        transformOrigin={transformOrigin}
        padding={PopupPadding.s}
        handleClose={onClose}
        closeIconSize={IconButtonSize.Small}
        role={isInteractive ? 'dialog' : 'status'}
        aria-live={isInteractive ? 'off' : 'polite'}
        aria-atomic={!isInteractive}
```

It just reads a little bit more clearly to me.